keepassxc-browser/content/fields.js

```diff
@@ -151,9 +165,7 @@ kpxcFields.getSegmentedTOTPFields = function(inputs, combinations) {
         || formLengthMatches(form)))) {
         // Use the form's elements
         addTotpFieldsToCombination(form.elements, exceptionFound);
-    } else if (inputs.length === DEFAULT_SEGMENTED_TOTP_FIELDS && inputs.every(i => (i.inputMode === 'numeric' && i.pattern.includes('0-9'))
-                || ((i.type === 'text' || i.type === 'number') && i.maxLength === 1)
-                || i.type === 'tel')) {
+    } else if (allAreSegmentedFields(inputs)) {
```

Comment:
Where is `inputs` defined?

Reply:
Maybe move the internal function out, it's a helper function and confusingly embedded in the middle of the parent one